operations/iuf/workflows/deploy_product.md

```diff
@@ -30,7 +30,7 @@ Once this step has completed:
 
 ## 2. Upgrade Kubernetes
 
-**`NOTE`** This subsection is optional and can be skipped if upgrading only CSM through IUF.
+**`NOTE`** This subsection is optional and mandatory only if Upgrading CSM manually and additional products with IUF
```

Comment:
I think this wording could be improved. It is confusing to say this is optional because it is required by the process, it may not be necessary to run manually though. Below is how I would suggest wording this note.

```suggestion
**`NOTE`** This subsection, `Upgrade Kubernetes`, is only mandatory if Upgrading CSM manually and additional products with IUF.
This subsection should **not** be executed if Upgrading CSM with IUF because this will automatically be executed as a hook script by IUF.
```